po/zh_TW.po

```diff
@@ -37,7 +37,7 @@ msgstr ""
 
 #: data/com.github.johnfactotum.Foliate.desktop.in:12
 msgid "Ebook;Book;EPUB;Viewer;Reader;"
-msgstr "Ebook;Book;EPUB;Viewer;Reader;電子書;書;檢視器;閱讀器;"
+msgstr "電子書;圖書;EPUB;檢視器;閱讀器"
```

Comment:
This is the list of keywords of the application. Should probably keep the English keywords, because they are often useful to many users. See for example https://github.com/elementary/calculator/issues/61. It's also the common practice at least among GNOME apps.

Also, the list _must_ end with a semicolon.

Reply:
Thanks for pointing out, I will resovle these soon :smile: